packages/amplify-cli/src/init-steps/s0-analyzeProject.js

```diff
@@ -95,20 +95,15 @@ async function getEnvName(context) {
   let envName;
 
   const isEnvNameValid = inputEnvName => {
-    let valid = true;
-
-    if (inputEnvName.length > 10 || inputEnvName.length < 2 || /[^a-z]/g.test(inputEnvName)) {
-      valid = false;
-    }
-    return valid;
+    return /^[a-z]{2,10}$/.test(inputEnvName);
   };
 
   if (context.exeInfo.inputParams.amplify && context.exeInfo.inputParams.amplify.envName) {
     if (isEnvNameValid(context.exeInfo.inputParams.amplify.envName)) {
       ({ envName } = context.exeInfo.inputParams.amplify);
       return envName;
     }
-    context.print.error('Environment name should be between 2 and 10 characters (only lowercase alphabets).');
+    context.print.error('Environment name must be between 2 and 10 characters, and lowercase only.');
```

Comment:
This could be moved a const as the same message is used in both places

Reply:
Fixed